rpmio/macro.c

```diff
@@ -1784,6 +1784,29 @@ rpmDefineMacro(rpmMacroContext mc, const char * macro, int level)
     return rc;
 }
 
+int rpmMacroIsDefined(rpmMacroContext mc, const char *n)
+{
+    int defined = 0;
+    if ((mc = rpmmctxAcquire(mc)) != NULL) {
+	if (findEntry(mc, n, 0, NULL))
+	    defined = 1;
+	rpmmctxRelease(mc);
+    }
+    return defined;
+}
+
+int rpmMacroIsCallable(rpmMacroContext mc, const char *n)
+{
```

Comment:
Wouldn't `rpmMacroIsParametric()` be a better name for this?

Reply:
That depends on whether one considers "parametric" to be a descriptive name. I find the term pretty horrible, especially as it's called "parameterized" in the docs and comments. I just cannot bring myself to add call a function rpmMacroIsParameterized()...

Reply:
Maybe `rpmMacroHasParameters()`?

Reply:
That seems to answer a different question (ie whether the macro was called with parameters or not).

Reply:
But okay, I'll update it to use rpmMacroIsParametric(), it's at least more in line with our internal terminology than "callable".